docs/source/interpolation.rst

```diff
@@ -181,7 +181,7 @@ argument:
    :lines: 72-73, 80
 
 By default the missing degrees of freedom (DoFs, the global basis function
-coefficients which could not be set) are zero:
+coefficients which could not be set) are left unchanged:
```

Comment:
This is not quite correct. It should read
```
In this case, the default the missing degrees of freedom (DoFs, the global basis function coefficients which could not be set) are zero:
```
and I should follow it up with an example which uses output=some_function to show in that case that the values are unmodified. If you leave this alone, I'll do that in a separate PR